src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -1180,61 +1168,48 @@ void SizeEvapCooler(EnergyPlusData &state, int const EvapCoolNum)
-    bool SizingDesRunThisZone;      // true if a particular zone had a Sizing:Zone object and zone sizing was done
-    Real64 PadAreaDes;              // Autosized celdek pad area for reporting
-    Real64 PadAreaUser;             // Hardsized celdek pad area for reporting
-    Real64 PadDepthDes;             // Autosized celdek pad depth for reporting
-    Real64 PadDepthUser;            // Hardsized celdek pad depth for reporting
+    bool IsAutoSize; // Indicator to autosize
 
     Real64 volFlowRateDes; // Autosized volume flow rate for reporting
     std::string CompType;  // for ease in getting objects
 
     // inits
-    CoolerOnOApath = false;
-    CoolerOnMainAirLoop = false;
-    IndirectVolFlowRateDes = 0.0;
-    IndirectVolFlowRateUser = 0.0;
-    PadAreaDes = 0.0;
-    PadAreaUser = 0.0;
-    PadDepthDes = 0.0;
-    PadDepthUser = 0.0;
+    bool CoolerOnOApath = false;
+    bool CoolerOnMainAirLoop = false;
+    Real64 IndirectVolFlowRateDes = 0.0;  // Autosized volume flow rate for reporting
+    Real64 IndirectVolFlowRateUser = 0.0; // Hardsized volume flow rate for reporting
+    Real64 PadAreaDes = 0.0;              // Autosized celdek pad area for reporting
+    Real64 PadAreaUser = 0.0;             // Hardsized celdek pad area for reporting
+    Real64 PadDepthDes = 0.0;             // Autosized celdek pad depth for reporting
+    Real64 PadDepthUser = 0.0;            // Hardsized celdek pad depth for reporting
 
     auto &CurSysNum(state.dataSize->CurSysNum);
     auto &CurZoneEqNum(state.dataSize->CurZoneEqNum);
     auto &FinalSysSizing(state.dataSize->FinalSysSizing);
     auto &EvapCond(state.dataEvapCoolers->EvapCond);
     auto &thisEvapCond(EvapCond(EvapCoolNum));
 
-    HardSizeNoDesRun = !((state.dataSize->SysSizingRunDone || state.dataSize->ZoneSizingRunDone));
+    bool HardSizeNoDesRun = !((state.dataSize->SysSizingRunDone || state.dataSize->ZoneSizingRunDone));
+    bool SizingDesRunThisAirSys = false; // true if a particular air system had a Sizing:System object and system sizing done
+    bool SizingDesRunThisZone = false;   // true if a particular zone had a Sizing:Zone object and zone sizing was done
 
     if (CurSysNum > 0) {
         CheckThisAirSystemForSizing(state, CurSysNum, SizingDesRunThisAirSys);
-    } else {
-        SizingDesRunThisAirSys = false;
+        if (SizingDesRunThisAirSys) {
+            HardSizeNoDesRun = false; // Check if design infomation is available
+        }
     }
     if (CurZoneEqNum > 0) {
         CheckThisZoneForSizing(state, CurZoneEqNum, SizingDesRunThisZone);
-    } else {
-        SizingDesRunThisZone = false;
-    }
-    if (SizingDesRunThisAirSys) {
-        HardSizeNoDesRun = false; // Check if design infomation is available
+        if (SizingDesRunThisZone) {
+            HardSizeNoDesRun = false; // Check if design infomation is available
+        }
```

Comment:
Here's a change I made that might be the cause of diffs. Originally HardSizeNoDesRun was only set for systems using SizingDesRunThisAirSys. Now it's set for zone equipment and air systems. Easy to test if this is the issue.